resources/views/themes/bimbala/about_us.blade.php

```diff
@@ -3,47 +3,48 @@
-                    <p class="text-xl text-purple-200">
+                    <h3 class="relative text-3xl tracking-tight sm:text-4xl text-black mt-4">Values</h3>
+                    <p class="text-xl text-black">
                         The first principle of our company is to be transparent. Honesty is something we value the most.
                         We are always aiming to show our customers the things as they are.
                     </p>
 
-                    <h3 class="relative text-3xl tracking-tight sm:text-4xl text-white mt-4">Target</h3>
-                    <p class="text-xl text-purple-200">
-                        The product is carried mostly on the shoulders of two university students. Our dream to succeed to address
-                        the needs of each business which carries about its customers' feedback. This includes easier work process for managers as well
-                        as all desperate employees from a support team.
+                    <h3 class="relative text-3xl tracking-tight sm:text-4xl text-black mt-4">Target</h3>
+                    <p class="text-xl text-black">
+                        The product is carried mostly on the shoulders of two university students.
+                         Our dream is to succeed in addressing the needs of each business that carries about its customers' feedback. 
+                         This includes easier work process for managers as well as all desperate employees from a support team.
                     </p>
                 </div>
             </div>
-            <div class="max-w-7xl mx-auto w-full grid grid-cols-9 px-2">
+            <div class="max-w-7xl mx-auto w-full grid grid-cols-9 px-2 ">
                 <!-- Stack 3 -->
             {{--<div class="col-span-4 w-full h-full ">
-                <div class="w-full h-full bg-white rounded-md p-2 md:pl-4">
+                <div class="w-full h-full bg-black rounded-md p-2 md:pl-4">
                     <h1 class="text-black text-2xl font-bold py-2">Title</h1>
                     <p class="text-gray-500 sm:text-md text-sm">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt corporis consequuntur voluptate nulla iusto quam ut quasi, eaque quas omnis vero totam ullam, reprehenderit ratione pariatur accusamus suscipit odit nostrum?</p>
                 </div>
             </div>
             <div class="relative col-span-1 w-full h-full flex justify-center items-center">
-                <div class="h-full w-1 bg-white"></div>
-                <div class="absolute w-8 h-8 rounded-full bg-white z-10 text-black text-center"></div>
+                <div class="h-full w-1 bg-black"></div>
+                <div class="absolute w-8 h-8 rounded-full bg-black z-10 text-black text-center"></div>
             </div>
             <div class="col-span-4 w-full h-full"></div>--}}
 
 
                 <!-- Stack 2 -->
```

Comment:
Let's darken this stack, taking into account the new design.

Reply:
I'm not sure I understand your suggestion. I wasn't really happy with how it turned out so If you have an idea of making it better I'm open to suggestions.

Reply:
My intention is adding more contrast here, so I think something like that would be fine:
![image](https://user-images.githubusercontent.com/14351652/180420358-1fabd584-7019-4af0-86c3-45849eae478e.png)
It only lacks of any images. If we find one, would be great.

Reply:
Just push it as you want. We can make a ticket for future reworking.